.github/workflows/release.yml

```diff
@@ -412,3 +412,92 @@ jobs:
           tags: ${{ steps.docker_meta.outputs.tags }}
           labels: ${{ steps.docker_meta.outputs.labels }}
           platforms: linux/amd64,linux/arm64
+
+  release-dashmate:
+    name: Release Dashmate packages
+    runs-on: ${{ matrix.os }}
+    strategy:
+      fail-fast: false
+      matrix:
+        os: [ubuntu-22.04, macos-12]
+    steps:
+      - name: Check out repo
+        uses: actions/checkout@v3
+
+      - name: Setup Node.JS
+        uses: actions/setup-node@v3
+        with:
+          node-version: '16'
+
+      - name: Install macOS build deps
+        if: runner.os == 'macOS'
+        run: |
+          brew install llvm docker colima
+          colima start
+          echo "/usr/local/opt/llvm/bin" >> $GITHUB_PATH
+
+      - name: Install Linux build deps
+        if: runner.os == 'Linux'
+        run: sudo apt-get install -y nsis
+
+      - uses: dtolnay/rust-toolchain@master
+        with:
+          toolchain: stable
+          target: wasm32-unknown-unknown
+
+      - name: Enable corepack
+        run: corepack enable
+
+      - name: Disable NPM audit
+        run: npm config set audit false
+
+      - name: Enable Yarn unplugged modules cache
+        uses: actions/cache@v3
+        with:
+          path: '.yarn/unplugged'
+          key: ${{ runner.os }}-yarn-unplugged-${{ hashFiles('yarn.lock') }}
+          restore-keys: |
+            ${{ runner.os }}-yarn-unplugged-
+
+      # Workaround for lack of yarn 3 support in oclif
+      - name: Patch package-json
+        run: |
+          echo "$(jq '. |= .+ { "dependencies": {"debug": "^4.3.4", "lodash": "^4.17.21", "strip-ansi": "^6.0.1", "fs-extra": "^8.1" },}' package.json)" > package.json
+
+      - name: Install dependencies
+        run: yarn install
+        env:
+          YARN_ENABLE_IMMUTABLE_INSTALLS: false
+
+      - name: Build packages
+        run: yarn build
+        env:
+          CARGO_BUILD_PROFILE: release
+      
+      - name: Generate manifest JSON
+        run: yarn workspace dashmate oclif manifest
+
+      - name: Pack tarballs
+        run: yarn workspace dashmate oclif pack tarballs
+        if: runner.os == 'Linux'
+
+      - name: Pack Linux deb
+        run: |
+          yarn install
+          yarn workspace dashmate oclif pack deb
+        if: runner.os == 'Linux'
+
+      - name: Pack Windows exe
+        run: |
+          yarn install
+          yarn workspace dashmate oclif pack win
+        if: runner.os == 'Linux'
+
+      - name: Pack macos pkg
+        run: yarn workspace dashmate oclif pack macos
+        if: runner.os == 'macOS'
```

Comment:
Shall we do matrix with three segments and add a variable with pack type?

Reply:
Not sure what you mean by this? The `runner.os` variable is always available, and if we check instead for e.g. `ubuntu-22.04` then we need to update all the conditions every time the runner base image changes.

Reply:
I meant to add one more variable for win/deb/macos

Reply:
If you define matrix like this:
```yaml
strategy:
   fail-fast: false
   matrix:
     include:
          - os: ubuntu-22.04
             package_type: win 
          - os: ubuntu-22.04
             package_type: deb
          - os: macos-12
             package_type: macos
```
The list of steps will be more understandable because you can get rid of additional conditions there.

Reply:
More understandable to who? I still don't understand what you want to do with these variables :joy: 

What is the benefit of doing three runs instead of two? If you are thinking of something like `yarn workspace dashmate oclif pack $matrix.package_type` then we would actually need four runs to build tarballs as well, right?

Reply:
Oh, yeah, let's do one more for tarballs as well. I mean the logic will be more straightforward: you need 4 packages for different OS and to create them you need `yarn workspace dashmate oclif pack $matrix.package_type`. So we reduce branching from steps and visually it will be more understandable as well.